client/my-sites/email/email-providers-stacked-comparison/provider-cards/get-upsell-props.ts

```diff
@@ -0,0 +1,20 @@
+import { translate } from 'i18n-calypso';
+import page from 'page';
+import type { MailboxListProps } from 'calypso/my-sites/email/form/mailboxes/components/new-mailbox-list';
+
+const getUpsellProps = ( {
+	isDomainInCart = false,
+	selectedSiteSlug,
```

Comment:
Shall we move the optional parameter to the end?

Reply:
For destructured parameters the order of the default values shouldn't matter.